tests/typecheck/core/test_cache.yml

```diff
@@ -0,0 +1,23 @@
+-   case: test_default_cache_exposes_base_cache_methods
```

Comment:
Thanks, testing things is great. But, this is a bit of an overkill.

We generally test dynamic / plugin related / complex generic code. Regular annotations are not tested: they will just make things harder.